course-v2/layouts/partials/get_instructors.html

```diff
@@ -4,17 +4,15 @@
   {{ $url := (print (strings.TrimSuffix "/" $staticApiBaseUrl) "/instructors/" . "/index.json") }}
   {{ with resources.GetRemote $url }}
     {{ with .Err }}
-      {{ $errorMessage := printf "Something went wrong while fetching instructors on %v via %v with error: %v" site.BaseURL $url .Err }}
```

Comment:
Because we're in a `with .Err` block, `.` refers to the error already. So we shouldn't be using `.Err`, we should be using just `.`.